test/custom.spec.js

```diff
@@ -99,6 +99,19 @@ describe("Rule: custom", function(){
         return testSucceeds([]);
     });
 
+    it.only("should provide file information", function(){
```

Comment:
```suggestion
    it("should provide file information", function(){
```

Reply:
leaving `only` on tests is something of a speciality of mine haha. i do it _all_ the time. i've fixed now, and squashed into the last commit, to avoid some noise in the commit history